classes/decision/types/BackToInternalReview.inc.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * @file classes/decision/types/BackToInternalReview.inc.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2000-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class decision
+ *
+ * @brief A decision to return a submission to the internal review stage from copy editing stage
+ */
+
+namespace APP\decision\types;
+
+use APP\decision\Decision;
+use APP\submission\Submission;
+use Illuminate\Validation\Validator;
+use PKP\context\Context;
+use PKP\decision\DecisionType;
+use PKP\decision\Steps;
+use PKP\decision\steps\Email;
+use PKP\decision\types\interfaces\DecisionRetractable;
+use PKP\decision\types\traits\InCopyEditing;
+use PKP\decision\types\traits\NotifyAuthors;
+use PKP\decision\types\traits\WithReviewAssignment;
+use PKP\mail\mailables\DecisionBackToInternalReviewNotifyAuthor;
+use PKP\security\Role;
+use PKP\submission\reviewRound\ReviewRound;
+use PKP\user\User;
+use PKP\submission\reviewRound\ReviewRoundDAO;
+use PKP\db\DAORegistry;
+
+class BackToInternalReview extends DecisionType implements DecisionRetractable
```

Comment:
Let's call this `BackToInternalReviewFromCopyediting` and call the one from external review `BackToInternalReview`. That will follow the pattern where "BackToXXXX" means back from the next stage. And if we are skipping a stage, then we add the "FromXXXX" part.

Reply:
I think making it `BackToXXXXXXXReviewFromCopyediting` makes better sense . But lets keep others as it is as those name give us better understanding of their purpose. Done making the changes to `BackToXXXXXXXReviewFromCopyediting` .